### PR TITLE
Added Hotel booking app

### PR DESCRIPTION
**Pull request by Ivanko-Mosiychuk**

### CODE REVIEW

File: src/main/java/com/hotel/room/getPlaces.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.hotel.room;
+
+public interface getPlaces {
```
Comment (olobanok) [inline src/main/java/com/hotel/room/getPlaces.java]: please rename interface

File: src/main/java/com/hotel/room/getPlaces.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    public int getNumberOfFreePlaces();
```
Comment (olobanok) [inline src/main/java/com/hotel/room/getPlaces.java]: public access modifier is redundant, methods in interface are abstract and public by default

File: src/main/java/com/hotel/room/Room.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package com.hotel.room;
+
+public class Room {
```
Comment (olobanok) [inline src/main/java/com/hotel/room/Room.java]: shouldn't Room class be abstract?

Comment (Ivanko-Mosiychuk) [reply src/main/java/com/hotel/room/Room.java]: Yes. It was my mistake. Fixed it.

File: src/main/java/com/hotel/room/getPlaces.java (hunk 2 above)
Comment (Ivanko-Mosiychuk) [reply src/main/java/com/hotel/room/getPlaces.java]: Okey, fixed it.

File: src/main/java/com/hotel/room/getPlaces.java (hunk 1 above)
Comment (Ivanko-Mosiychuk) [reply src/main/java/com/hotel/room/getPlaces.java]: Done

**Pull request closed without merging**